pkgs/by-name/lu/lua-language-server/package.nix

```diff
@@ -28,36 +34,49 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
-    CoreFoundation
-    Foundation
-    ditto
+    # aligned_alloc
```

Comment:
What's up with alligned_alloc?

Reply:
Reason why we have to target a non default sdk